pom.xml

```diff
@@ -2120,9 +2119,4 @@
       </plugin>
     </plugins>
   </reporting>
-
-  <prerequisites>
```

Comment:
I am not familiar with this. Why did we have both and why do we not need both now? If it works, then I guess I don't care too much. Just curious.

Reply:
Prerequisites is only used for maven-plugins, and since maven 3.5.0 users get a warning to avoid using it.
Ref. https://maven.apache.org/docs/3.5.0/release-notes.html